source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+---
+id: add-new-attribute
+title: Add a new attribute
+---
+
+Let's say we want to add a new element to display on a product's page (for instance, a product's rate). You will have to:
+
+1. Add the attribute's actual value to your database.
+2. Create a new module.
+3. Define the schema, implement the resolver and display it on screen.
+
+## Add a value to your database
+
+The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [this](https://www.fastcomet.com/tutorials/magento2/product-attributes).
```

Comment:
```suggestion
The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [How to Create Product Attributes in Magento 2(https://www.fastcomet.com/tutorials/magento2/product-attributes).
```

Links saying stuff like `click here` are not accessible to everyone. Stating exactly what the link is about within the link will improve accessibility and make things more explicit.